### PR TITLE
Refresh network-denyall-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-denyall-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 1]
```diff
@@ -1,209 +1,700 @@
 ---
 id: network-denyall-snmp
-title: DenyAll SNMP
+title: DenyAll SNMP (Rohde & Schwarz)
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 2]
```diff
@@ -1,209 +1,700 @@
...
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Denyall-SNMP-custom" label="Net-Denyall-SNMP-custom">
+
+| Alias   | Modèle de service               | Description                       |
+|:--------|:--------------------------------|:----------------------------------|
+| Cpu     | Net-Denyall-Cpu-SNMP-custom     | Contrôle l'utilisation processeur |
+| Load    | Net-Denyall-Load-SNMP-custom    | Contrôle la load average          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md:31]:
```suggestion
| Load    | Net-Denyall-Load-SNMP-custom    | Contrôle la charge moyenne          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 3]
```diff
@@ -1,209 +1,700 @@
...
+| Memory  | Net-Denyall-Memory-SNMP-custom  | Contrôle la mémoire               |
+| Storage | Net-Denyall-Storage-SNMP-custom | Contrôle l'espace disque          |
+| Swap    | Net-Denyall-Swap-SNMP-custom    | Contrôle la swap                  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md:34]:
```suggestion
| Swap    | Net-Denyall-Swap-SNMP-custom    | Contrôle le swap                  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 4]
```diff
@@ -1,209 +1,700 @@
...
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte  **Net-Denyall-SNMP-custom** |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                    | Description                                                             |
+|:-----------------------------------|:------------------------------------------------------------------------|
+| Net-Denyall-SNMP-Interface-Name    | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md:67]:
```suggestion
| Net-Denyall-SNMP-Interface-Name    | Découvre les interfaces réseau et en supervise le statut et l'utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 5]
```diff
@@ -1,209 +1,700 @@
...
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
 
 </TabItem>
 </Tabs>
 
 ## Prérequis
 
-Afin de contrôler votre équipement DenyAll, le SNMP doit être configuré. 
+### Configuration SNMP
+Afin de superviser votre **DenyAll** en SNMP, il est nécessaire de configurer l'agent sur l'équipement.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md:169]: l'agent?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 6]
```diff
@@ -1,209 +1,700 @@
...
+| UNITS          | Units of thresholds (default: '%') ('%', 'B')                                                                                                    | %                            |             |
+| FILTERSTORAGE  | Set the storage (number expected) example: 1, 2,... (empty means 'check all storage')                                                            |                              |             |
+| WARNINGACCESS  | Warning threshold                                                                                                                                |                              |             |
+| CRITICALACCESS | Critical threshold. Check if storage is readOnly: --critical-access=readOnly                                                                     |                              |             |
+| WARNINGCOUNT   | Warning threshol                                                                                                                                 |                              |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md:378]:
```suggestion
| WARNINGCOUNT   | Warning threshold                                                                                                                                 |                              |             |
```

File: pp/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 7]
```diff
@@ -1,210 +1,699 @@
...
+* **Net-Denyall-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Denyall-SNMP-custom" label="Net-Denyall-SNMP-custom">
+
+| Service Alias | Service Template                | Service Description |
+|:--------------|:--------------------------------|:--------------------|
+| Cpu           | Net-Denyall-Cpu-SNMP-custom     | Check cpu usage     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-denyall-snmp.md:29]:
```suggestion
| Cpu           | Net-Denyall-Cpu-SNMP-custom     | Check CPU usage     |
```

File: pp/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 8]
```diff
@@ -1,210 +1,699 @@
...
+| Load          | Net-Denyall-Load-SNMP-custom    | Check load-average  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-denyall-snmp.md:30]:
```suggestion
| Load          | Net-Denyall-Load-SNMP-custom    | Check load average  |
```

File: pp/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 9]
```diff
@@ -1,210 +1,699 @@
...
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
 
 </TabItem>
 </Tabs>
 
 ## Prerequisites
 
-To control your DenyAll, the SNMP must be configured.
+### SNMP Configuration
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-denyall-snmp.md:167]: missing sentence

File: pp/integrations/plugin-packs/procedures/network-denyall-snmp.md [hunk 10]
```diff
@@ -1,210 +1,699 @@
...
+To use this pack, the SNMP service must be properly configured on your **DenyAll** equipment.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-denyall-snmp.md:168]:
```suggestion
To use this pack, the SNMP service must be properly configured on your **DenyAll** respurce.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md (hunk 5 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-denyall-snmp.md:169]:
```suggestion
### Configuration SNMP

L'agent SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**